### ISSUE TITLE
How do you spawn floating windows?

### ISSUE THREAD

**Issue by jkimmerling** [leftwm/leftwm#251]
I know I am probably missing something really easy, but I cannot figure out how to spawn specific windows in floating mode.

I want to spawn a floating terminal window, but I cannot find out how to do this. I've googled, searched here, looked through all the example themes, etc. No luck lol.

**Comment by mautamu** [comment]
There isn't, to the best of my knowledge, a way to spawn something as floating (I think this depends on #165). However to make an existing window floating, modkey + right/left click-drag will turn a window into a floating window.

**Comment by jkimmerling** [comment]
Well darn, I was hoping to be able to have polybar spawn floating terminals for things like wttr.in weather reports and such.

**Comment by mautamu** [comment]
If you can choose what hints Polybar puts on a window, Dialog, Splash (pending #184), and Floating all float. Toolbar might too?

**Comment by jkimmerling** [comment]
Could you possibly give me the syntax on how to choose what hints something gets?

And it would be a very nice feature for ephemeral windows such as weather reports or memo programs :)

**Comment by jkimmerling** [comment]
I cannot find a way to call the terminal in question (kitty) as any of those "hints"

I tried setting the class, title, and name to all three of the ones you mentioned, and nothing worked.

Is there a way to call a program from the terminal and set the tag? As the polybar spawns are just shell commands

**Comment by AethanFoot** [comment]
If you are creative enough you may be able to manipulate rofi to display what you want, as is can be display anywhere on the screen.

**Comment by jkimmerling** [comment]
Fair enough, I do have some examples to play with for that. It would just be nice to have a flag or something that could be added to a call to make it float. But I think that would be way down on the priority list, or even not wanted, since it seems they want to keep it as simple as possible. Though it would make sense to allow themes to easily push this sort of feature.

**Comment by jkimmerling** [comment]
it seems Guake does not float either. It seems I will have to do something "creative" :(

**Comment by hertg** [comment]
I'm closing this because it should be resolved with window rules in #631
